controlplane/pkg/plugins/utils.go

```diff
@@ -38,6 +39,32 @@ func StartPlugin(name, registry string, services map[plugins.PluginCapability]in
 		return err
 	}
 
+	go func() {
+		registered := true
+		for ; ; time.Sleep(5 * time.Second) {
+			resp, err := http.Get("http://localhost:5555/liveness")
+			if err != nil {
+				logrus.Errorf("failed to get NSMD liveness: %v", err)
+			}
+
+			nsmdAlive := err == nil && resp.StatusCode >= 200 && resp.StatusCode < 400
```

Comment:
In this case, there would be nested 'if registered {}' to output "NSMD failure detected.." message. which complicates readability imho

Reply:
can we simply use log.Error("....", status Code, err, registered)... and not use nested if blocks?

Reply:
Then It will report to log every 'check period' which might be a lot of text, is it Ok?
Currently, 'NSMD failure detectedNSMD failure detected' is output only once

Reply:
It will not report every 'check period', It will report only if we have error or response status is not 200

Reply:
Sorry, I meant it will report every 'check period' during nsmd failure, like 5 minutes, for example.
is it Ok?

Reply:
maybe we could use `logrus.Fatal` after timeout in 5 minutes? What do you think?

Reply:
Fatal means we should terminate monitoring gorutine after 5 minutes of inaccessible nsmd?
Are there some benefits for implementing such a complicated behavior?

Reply:
My point is that we could keep it simple. 
I mean the checks at lines 114 and 122.
This is minor comment and can be ignored.